src/components/SlateEditor/plugins/aside/SlateRightAside.jsx

```diff
@@ -30,6 +31,12 @@ const SlateRightAside = props => {
       <Button stripped onClick={onRemoveClick} {...classes('delete-button')}>
         <Cross />
       </Button>
+      <Button
```

Comment:
Should at least have a title on the button to help the user? Or use Tooltip from ndla-ui